import-automation/import-progress-dashboard-frontend/client/.gitignore

```diff
@@ -0,0 +1,26 @@
+.vscode
```

Comment:
consider merging this with the .gitignore in the parent dir, there's only one or two additional filetypes

Reply:
removed. this one